masterlist.yaml

```
@@ -6894,8 +6896,10 @@ plugins:
 
   - name: 'RealisticNeedsandDiseases.esp'
     url:
-      - 'https://www.nexusmods.com/skyrimspecialedition/mods/3487'
-      - 'https://www.nexusmods.com/skyrimspecialedition/mods/23799'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/3487/'
+        name: 'Realistic Needs and Diseases All-In-One for USSEP'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/23799/'
+        name: 'Realistic Needs and Diseases 2.0'
```

There's also [iWant RND for CACO](https://www.nexusmods.com/skyrimspecialedition/mods/33227), I didn't get around to adding it yet.

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/3487/'
        name: 'Realistic Needs and Diseases All-In-One for USSEP'
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/23799/'
        name: 'Realistic Needs and Diseases 2.0'
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/33227/'
        name: 'iWant RND for CACO'
```